### PR TITLE
Support for tables in SimpleHtml

### PR DESCRIPTION
**Pull request by ltatarev**
Feature adds support for tables in `SimpleHtml` component using `react-native-render-html-table-bridge` package.
Table rendering is handled really well, and in case of column overflow table is rendered as horizontal scroll view.

Few issues I have with current implementation are mentioned in ➡️ [this PR](https://github.com/shoutem/mobile/pull/753).

Currently, the table is styled as shown in screenshot in above mentioned PR.

✍️  **EDIT**: Added support for rendering image attachments using `Image` component. Styling was added to keep consistent style to `Html` `Img` component.

### CODE REVIEW

**Review by Definitely-Not-Vlad — APPROVED** [review]
Great job!

**Review by majaklajic — APPROVED** [review]

**Review by majaklajic — APPROVED** [review]
Great job, Lucice!

File: html/components/SimpleHtml.js [hunk 1]
```diff
@@ -233,10 +233,13 @@ class SimpleHtml extends PureComponent {
...
-      ...defaultTableStylesSpecs,
-      ...style.table,
-    });
+    const tableStyle = _.get(style, 'table', {});
+    const tableCssStyle = _.get(style, 'tableCss', '');
+    const cssRules =
+      cssRulesFromSpecs({
+        ...defaultTableStylesSpecs,
+        ...tableStyle,
+      }) + tableCssStyle;
```
Comment (tomislav-arambasic) [inline html/components/SimpleHtml.js:242]: Maybe like this?

```
const stylesCssString = cssRulesFromSpecs({
        ...defaultTableStylesSpecs,
        ...tableStyle,
      });
   const cssRules = `${stylesCssString}${tableCssStyle}`;
```

Seems easier to read, but up to you.
You've already got approvals, merge when you want

Comment (ltatarev) [reply html/components/SimpleHtml.js]: Good point, agree that it's easier to read.
I renamed `stylesCssString` to just `cssStyle` so it's shorter and similar name to `tableCssStyle`, hope that's alright.

**Pull request merged**